index.js

```diff
@@ -43,7 +43,16 @@ var excludedKeys = {
 	$webkitStorageInfo: true,
 	$window: true
 };
+var isChromeExtension = (function () {
+	/* global chrome */
+	return (typeof chrome !== 'undefined') &&
+		(typeof chrome.runtime !== 'undefined') &&
+		(typeof chrome.runtime.id !== 'undefined');
+}());
 var hasAutomationEqualityBug = (function () {
+	if (isChromeExtension) {
+		return false;
```

Comment:
I appreciate the concern, and in general that's what all the explicit exceptions are for - but in this case, many of the problematic ones are the very ones that need testing.

I have a hunch - does overwriting the console methods allow you to intercept the warnings issued in this case?